package.json

```diff
@@ -42,6 +42,8 @@
   },
   "dependencies": {
     "amazon-cognito-identity-js": "^3.2.0",
+    "aws-sdk": "^2.578.0",
+    "dynamodb-data-types": "^3.0.1",
```

Comment:
`aws-sdk` chyba można przenieść do devDependecies
`dynamodb-data-types` można usunąć

Reply:
aws-sdk standardowo jest w devDependencies, to drugie usuniete